cve_bin_tool/checkers/apache.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+"""
+CVE checker for Apache HTTP Server
+
+https://www.cvedetails.com/vulnerability-list/vendor_id-45/product_id-66/Apache-Http-Server.html
+"""
+from cve_bin_tool.checkers import Checker
+
+
+class ApacheHttpServerChecker(Checker):
+    CONTAINS_PATTERNS = [
+        r"logs/apache_runtime_status",
+    ]
+    FILENAME_PATTTERN = [r"httpd"]
+    VERSION_PATTERNS = [
+        r"Apache/([0-9]+\.[0-9]+(\.[0-9]+)?)",
```

Comment:
On 2nd look this might not be a unique pattern. This pattern could possibly exist in other Apache products and could lead to false positives.

Reply:
I found this one:
```httpd-2.4.52-1.fc35.x86_64.debug```

I don't know how Fedora-specific this is though, maybe Ubuntu etc have a similar package name embedded in the binary?

Reply:
Perhaps we can also see other patterns in the string to make sure it's httpd specifically and not any other Apache product?

Reply:
Patterns ending with `.debug` are often considered not optimal. You can look for multi-line strings that are unique to the package too (like in #1200).

Reply:
I added more patterns in CONTAINS_PATTERNS to be more HTTP server specific, and I wasn't able to find better versioning information in the strings than the one currently in the code.